cves/kernel/CVE-2021-4032.yml

```diff
@@ -408,17 +436,20 @@ lessons:
     applies:
     note:
   least_privilege:
-    applies:
-    note:
+    applies: True
+    note: |
+      giving all users as little privilege as possible will would lower the amount of people able to 
+      casue the denial of service because to do it you need special user permissions.
   frameworks_are_optional:
     applies:
     note:
   native_wrappers:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: True
+    note: |
+      distrusting the given VCPU memory given by adding and changing boolean checks is sort distrusting inputs within the system itself. 
```

Comment:
I agree with this, I would just delete the word "sort" from this sentence as it is a little confusing to read.

Reply:
i would split this to the next line as it is a little long for readability

Reply:
thank you for feedback!